snapcraft/plugins/kernel.py

```diff
@@ -157,6 +153,23 @@ def enable_cross_compilation(self):
             self.project.kernel_arch))
         self.make_cmd.append('CROSS_COMPILE={}'.format(
             self.project.cross_compiler_prefix))
+        self._set_kernel_targets()
```

Comment:
I'm confused here. Why do you call it again if you already called it in the init?